File: geek-university-algoritmo-e-logica-de-programacao/c/exercicio_10.c
#include <stdio.h>

int main() {
	float valor_por_hora, salario;
	int horas_trabalhadas;

	printf("Qual valor você ganha por hora? ");
	scanf("%f", &valor_por_hora);

	printf("Quantas horas você trabalhou neste mes? ");
	scanf("%d", &horas_trabalhadas);

	salario = (horas_trabalhadas * valor_por_hora);

	printf("Neste mes você vai receber R$ %.2f", salario);
}

File: geek-university-algoritmo-e-logica-de-programacao/java/exercicio_10.java
import java.util.Scanner;

public class exercicio_10 {

	public static void main(String[] args) {
		float valor_por_hora, salario;
		int horas_trabalhadas;
		Scanner teclado = new Scanner(System.in);

		System.out.print("Qual o valor voce ganha por hora? ");
		valor_por_hora = teclado.nextFloat();
		
		System.out.print("Quantas horas voce trabalhou neste mes? ");
		horas_trabalhadas = teclado.nextInt();

		salario = (horas_trabalhadas * valor_por_hora);

		System.out.println("Neste mes voce vai receber R$ " + salario);
		
		teclado.close();
	}

}
